source/version.f90

```diff
@@ -1,7 +1,7 @@
 module mod_version
   ! Keep data type in sync with 'cr_version' and 'cr_moddate'
-  character(len=8),  parameter :: version = "5.2.2"
-  integer,           parameter :: numvers = 50202
-  character(len=10), parameter :: moddate = "08.04.2019"
+  character(len=8),  parameter :: version = "5.2.3"
+  integer,           parameter :: numvers = 50203
+  character(len=10), parameter :: moddate = "XX.04.2019"
```

Comment:
`XX.` ?

Reply:
Yes? I don't know the date until I click merge. I push that final commit then.